kustomize/go.mod

```diff
@@ -1,13 +1,12 @@
-module sigs.k8s.io/kustomize/kustomize
+module sigs.k8s.io/kustomize/kustomize/v3
 
 go 1.13
 
 require (
-	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
-	github.com/googleapis/gnostic v0.3.0 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	sigs.k8s.io/kustomize/api v0.1.1
+	sigs.k8s.io/kustomize/kustomize v0.0.0-20191024000301-ce7ebe3299dd
```

Comment:
Why does this module require itself?

Reply:
good eye, thanks for catching that.  My recursive sed omitted main.go.  Fixed.